example/config.js

```diff
@@ -7,12 +7,20 @@ var System = (function() {
   // Create instance of bitimports
   var importer = bitimports.config({
     "paths": {
-      "babel": "../node_modules/babel-bits/dist/index.min.js",
-      "sass": "../node_modules/sassy-bits/dist/index.min.js",
-      "plugin1": "./js/plugin1.js"
-    }
+      "babel": "/node_modules/babel-bits/dist/index.min.js",
+      "sass": "/node_modules/sassy-bits/dist/index.min.js",
```

Comment:
I have removed sassy-bits and now we are just loading the css directly into the DOM.